src/renderer/box_drawing/renderer.rs

```diff
@@ -0,0 +1,1529 @@
+            Shade::Medium => 2.0,
+            Shade::Dark => 3.0,
+        };
+        let mut fg = self.fg_paint();
+        fg.set_style(PaintStyle::Fill);
+        fg.set_anti_alias(true);
+        match color_mode {
+            ColorMode::Normal => (),
+            ColorMode::Inverted => {
+                self.canvas.draw_paint(&fg);
+                fg.set_color(self.color_bg);
+            }
+        }
+
+        {
+            let stripe_sz = (3.0 * tile_sz.width.max(tile_sz.height), stripe_height);
+            match mirror {
+                MirrorMode::Normal => (),
+                MirrorMode::Mirror => {
+                    rotation_degrees = 180.0 - rotation_degrees;
+                    self.canvas.translate((tile_sz.width, 0.0));
+                }
+            };
+            let top_left = self.bounding_box.min;
+            for i in -1..NUM_STRIPES + 1 {
+                let (dx, dy) = (0., i as f32 * stripe_gap);
+                let stripe_top_left = top_left.translate(Vector2::new(dx, dy));
+                self.canvas.save();
+                self.canvas
+                    .rotate(rotation_degrees, Some(stripe_top_left.to_tuple().into()));
+                self.canvas.draw_rect(
+                    Rect::from_point_and_size(stripe_top_left.to_tuple(), stripe_sz),
+                    &fg,
+                );
+                self.canvas.restore();
+            }
+        }
+        self.canvas.restore();
+    }
+
+    fn triangle_path(&self, corner: Corner) -> Path {
+        let mut path = Path::default();
+        let bb = to_skia_rect(&self.bounding_box);
+        let top_left = (bb.left, bb.top);
+        let top_right = (bb.right, bb.top);
+        let bottom_left = (bb.left, bb.bottom);
+        let bottom_right = (bb.right, bb.bottom);
+        match corner {
+            Corner::TopLeft => {
+                path.move_to(top_left);
+                path.line_to(top_right);
+                path.line_to(bottom_left);
+            }
+            Corner::TopRight => {
+                path.move_to(top_right);
+                path.line_to(top_left);
+                path.line_to(bottom_right);
+            }
+            Corner::BottomRight => {
+                path.move_to(bottom_right);
+                path.line_to(top_right);
+                path.line_to(bottom_left);
+            }
+            Corner::BottomLeft => {
+                path.move_to(bottom_left);
+                path.line_to(top_left);
+                path.line_to(bottom_right);
+            }
+        }
+        path.close();
+        path
+    }
+
+    // draws a "rectircle" using the formula
+    //
+    // (|x| / a) ^ (2a / r) + (|y| / b) ^ (2b / r) = 1
+    // where 2a = width, 2b = height and r is radius
+    //
+    // See: https://math.stackexchange.com/questions/1649714/whats-the-equation-for-a-rectircle-perfect-rounded-corner-rectangle-without-s
+    fn draw_rounded_corner(&self, corner: Corner) {
```

Comment:
I think this is to complex, and probably slow. We only need to round the corners and  not create a singe formula for the whole rectangle. That is draw a 45 degree arc between the lines, so that the tangents match on both sides and the lines start the arc radius away from the corner.